test/kmeans.jl

```diff
@@ -32,6 +32,15 @@ end
     end
 end
 
+@testset "k=1 and k=n corner cases" begin
+    Random.seed!(34568)
+    @test length(kmeans(randn(2,3), 1).centers) == 2
+    @test length(kmeans(randn(2,3), 3).centers) == 6
```

Comment:
I think we don't really need the randomized examples here, since we are checking the fixed one.